lance-spark-34/src/test/resources/example_db/test_dataset1.lance/_transactions/4-dffa23f0-c357-4935-a9c1-e286099b5533.txn

```diff
@@ -0,0 +1,3 @@
+$dffa23f0-c357-4935-a9c1-e286099b5533�8
```

Comment:
have to paste test resources multiple times due to error:

```
[ERROR] Errors: 
[ERROR]   Spark34ConnectorReadTest>SparkConnectorReadTestBase.setup:55 » IO LanceError(IO): Operation not supported: Unsupported URI scheme: jar in url jar:file:/Users/zhaoqinye/oss/lance-spark/lance-spark-base/target/lance-spark-base-0.25.2-tests.jar!/example_db/test_dataset1.lance, /Users/zhaoqinye/oss/lance/rust/lance-io/src/object_store.rs:1029:27
```

Need to fix object_store.rs before using the same resource

Reply:
Actually I can configure test resource path, let me do that